src/desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -116,12 +118,27 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
     $element.addClass 'is-loading'
     action().finally => $element.removeClass 'is-loading' unless @comboForm
 
+  handleCheckTerms: (e) ->
+    console.log('checking terms')
+    if @$acceptTerms.prop('checked')
+      @enableSubmit()
+    else
+      @disableSubmit()
+  
+  disableSubmit: -> @$submit.addClass('is-disabled')
+  enableSubmit: -> @$submit.removeClass('is-disabled')
+
   onSubmit: =>
-    analyticsHooks.trigger 'registration:submit-address'
-    @loadingLock @$submit, =>
-      (if @validateForm() then Q() else Q.reject('Please review the error(s) above and try again.')).then =>
-        Q.all [@savePhoneNumber(), @tokenizeCard()]
-      .catch (error) =>
-        @showError error
-      .then =>
-        @trigger('submitted')
+    # @handleCheckTerms() # Shouldn't be necessary
+    if @$submit.hasClass('is-disabled') or !@$acceptTerms.prop('checked')
+      @disableSubmit()
+      @showError 'You must accept the Conditions of Sale.'
+    else
```

Comment:
My one thought in this conditional is that it might not be clear- the button being disabled *means that and only that* the conditions box is not checked. We redo disable submit in case for some reason the box is not checked but the button was not disabled. Likewise the commented-out line of code on 132 should be redundant.